README.md

```diff
@@ -0,0 +1,20 @@
+# example-sentence-extraction
+Module: Finding example sentences for keywords
+This module is responsible for fetching sentences containing the target user-inputed keyword, and by comparison with one another, select no more than 5 sentences which demonstrate the appropriate usage of the keyword.
+
+## Functional Design:
+External Package: MetaPy, NLTK
+- driver function loading json file into a dictionary
```

Comment:
What is the dictionary for? Can you improve your function description's clarity?